.travis.yml

```diff
@@ -94,7 +95,7 @@ matrix:
           astyle
       before_script:
       script:
-      - scripts/style-checker.sh check
+      - scripts/style-checker.sh diff
```

Comment:
Not sure when it was discarded, it was accepted on the meetings that I have
attended.

I still think it is a very useful feature. Giving me a red exclamation mark
without any detailed reason can put me off. Copyright checker also gives
hints about the fixes. gcc gives a hint about what an error means, it
doesn't just say that one of the specified files is in error.

On Tue, Sep 6, 2016 at 12:56 PM, Noémi Ványi <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/balabit/syslog-ng/pull/1194#discussion_r77613335>:
>
> > @@ -94,7 +95,7 @@ matrix:
> >            astyle
> >        before_script:
> >        script:
> > -      - scripts/style-checker.sh check
> > +      - scripts/style-checker.sh diff
>
> First we thought it was a good idea. But than, it was discarded.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/1194/files/13268b5522cb402e8d1464c73b2f9972e5d4cae6..9dad1cb2d4e02d302fbc9a7bffc772606179235d#r77613335>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOobpNsLDwv3IH9r5-PpodL8vUk4gyt0ks5qnUbogaJpZM4J0Jgy>
> .
>